commit 6d70a28bacfbe14f108420644dc4e04f7d9059a3 (refs/changes/10/2510/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-09-07T18:43:26+00:00 (2 years, 1 month ago)
    
    MORAY-426 Accept the "offset" option as a string
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/schema.js b/lib/schema.js
index f7fe64f..b1904e6 100644
--- a/lib/schema.js
+++ b/lib/schema.js
@@ -178,7 +178,8 @@ AJV_ENV.addSchema({
         },
         'offset': {
             description: 'An offset into a query result set',
-            type: 'number',
+            type: [ 'integer', 'string' ],
+            pattern: '^[0-9]+$',
             minimum: 0
         },
         'sort': {
